CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,38 @@
+# The Amethyst Code of Conduct
+
+## Conduct
+
+**Contact**: [amethyst-mods@amethyst-engine.org](mailto:amethyst-mods@amethyst-engine.org)
+
+* We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
+* On Discord, the forums, IRC, GitHub, or any other community area, please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
+* Please be kind and courteous. There's no need to be mean or rude.
+* Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
+* Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+* We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the <a href="http://citizencodeofconduct.org/">Citizen Code of Conduct</a>; if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+* Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact one of the channel ops or any of the [Amethyst moderation team][mod_team] immediately. Whether you're a regular contributor or a newcomer, we care about making this community a safe place for you and we've got your back.
+* Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+
+## Moderation
+
+
+These are the policies for upholding our community's standards of conduct. If you feel that a thread needs moderation, please contact the [Amethyst moderation team][mod_team].
+
+1. Remarks that violate the Amethyst standards of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner.)
+2. Remarks that moderators find inappropriate, whether listed in the code of conduct or not, are also not allowed.
```

Comment:
That is really vague and easily abusable imo

Reply:
To some extent, it's supposed to be that way. Being somewhat unspecific about it means that people will be a bit more careful when they interact with others. Of course, that falls apart quickly if the moderation team isn't any good but I don't see that becoming a problem in Amethyst anytime soon.

Reply:
Having moderated large communities before, creating dozens of micro-rules just gives rules lawyers things to argue about.

Reply:
A good team of moderators will not abuse this, and if one does, the rest of the moderation team will deal with it immediately. What I've seen in the past is people using "But it isn't in the code of conduct!" as an excuse, and then every possible inappropriate thing gets added to a huge list.

I'm open to re-phrasing that part, but will need some suggestions.

Reply:
I'd rephrase this to "Remarks that moderators find inappropriate, whether listed in the code of conduct or not, will be subject to the following:"

I think the phrase "not allowed." implies harsher punishment than intended.  We detail what exactly the consequences are pretty much right away, but we kind of get off on the wrong foot here.